docs/DeveloperGuide.md

```diff
@@ -179,7 +179,7 @@ The `Logic` component consists of:
 
 - `Parser`: Handles user input and decides the `Command` object to create.
 - `RecipeCommandParser`: When `Parser` detects that a `RecipeXYZComand` object needs to be created, the user input is handed over from `Parser` and will decide the `Command` object to create.
-- `Command`: An abstract class that defines the blueprint for the derived `*Command` classes. <a name="designListOfCommands"></a>
+- `Command`: An abstract class that defines the blueprint for the derived `XYZCommand` classes. <a name="designListOfCommands"></a>
```

Comment:
For `RecipeCommandParser` line, change `Command` to `RecipeXYZCommand` and help fix the type for `RecipeXYZCommand`.